src/TestInfrastructure/Framework/ConfigurableConnectSection.cs

```diff
@@ -5,14 +5,53 @@
 // </copyright>
 //-----------------------------------------------------------------------
 
+using SonarLint.VisualStudio.Integration.Progress;
 using SonarLint.VisualStudio.Integration.TeamExplorer;
-using System.Windows;
+using SonarLint.VisualStudio.Integration.WPF;
+using System.Windows.Input;
 
 namespace SonarLint.VisualStudio.Integration.UnitTests
 {
     internal class ConfigurableConnectSection : IConnectSection
```

Comment:
Did we not want to also rename this class (and interface) to match the concrete `SectionController`? I.e., `ConfigurableSectionController` and `ISectionController`.